docs/_static/css/light.css

```diff
@@ -0,0 +1,30 @@
+/* links */
+
+.rst-content a {
+    color: #002fa7;
+}
+
+.rst-content .highlighted {
+	background: #eac545;
+}
+
+/* inline code highlights */
+.xref,
+.py-meth,
+.rst-content code {
```

Comment:
Here I meant to change the color of the code blocks, as in the background color in the example contracts (currently a greenish yellow), not the inline code highlights. Sorry if that wasn't clear!

Reply:
Oh ok, I will fix these things and make a commit then..

Reply:
Was unable to resolve the **version issue where 'v' was missing**. Seems to be existing. I think we need to pass the version as v0.8.8 in the version variable (which I think is the **CMakeLists.txt**)  file.